client/appStore.go

```diff
@@ -0,0 +1,158 @@
+package client
+
+import (
+	"errors"
```

Comment:
Whats with the tabs?

Reply:
go fmt 🤷 , the go ecosystem is really strict about everything being formatted a certain way